ddtrace/debugging/_expressions.py

```diff
@@ -183,9 +180,31 @@ def _compile_direct_operation(ast):
             raise ValueError("Invalid argument: %r" % arg)
         return [Instr("LOAD_CONST", len)] + value + [Instr("CALL_FUNCTION", 1)]
 
+    if _type == "ref":
+        if not isinstance(arg, str):
+            return None
+
+        if arg == "@it":
+            return [Instr("LOAD_FAST", "_dd_it")]
+
+        return [
+            Instr("LOAD_FAST", "_locals"),
+            Instr("LOAD_CONST", arg),
+            Instr("BINARY_SUBSCR"),
+        ]
+
     return None
 
 
+def _compile_identifier(ast):
+    # type: (DDASTType) -> Optional[List[Instr]]
+    # identifier  =>  [a-zA-Z][a-zA-Z0-9_]*
+    if not isinstance(ast, str) or not IDENT_RE.match(ast):
+        return None
+
+    return [Instr("LOAD_ATTR", ast)]
```

Comment:
Just a note for follow-up work: it's probably best to move the `LOAD_ATTR` outside of this rule, as strictly speaking this is not how the identifier rule should work. This is fine for now because this rule is only used by the `getmember` operation.

Reply:
refactored